### PR TITLE
Documentation for new customize format.

### PR DESCRIPTION
**Pull request by andrey-git**
**Description:**

Documentation for new customize format.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#5215

### CODE REVIEW

File: source/getting-started/customizing-devices.markdown [hunk 1]
```diff
@@ -11,6 +11,8 @@ footer: true
 
 By default, all of your devices will be visible and have a default icon determined by their domain. You can customize the look and feel of your front page by altering some of these parameters. This can be done by overriding attributes of specific entities.
 
+`customize` consists of a list of attribute customization blocks
```
Comment (kellerza) [inline source/getting-started/customizing-devices.markdown]: I wonder if it makes sense to keep the old mode around? Maybe something to ask in the chat channel?

Comment (andrey-git) [inline source/getting-started/customizing-devices.markdown]: Paulus said: "yeah, top of the introduction the new format, maybe a section with old format at the bottom ?"

**Comment by andrey-git** [comment]
The parent PR has been merged.

**Comment by fabaff** [comment]
Will be merged after the release of 0.37. I'm unable to change the branch as the PR is pointing to `current` instead of `next`.

**Comment by balloob** [comment]
@fabaff actually GitHub introduced the option so that that is possible 👍

**Pull request merged**